examples/serverless/README.md

```diff
@@ -0,0 +1,18 @@
+# Serverless - AWS Node.js Typescript
+
+This project has been generated using the `aws-nodejs-typescript` template from the [Serverless framework](https://www.serverless.com/).
+
+For detailed instructions, please refer to the [documentation](https://www.serverless.com/framework/docs/providers/aws/).
+
+## Test your service
+
+This template contains a single lambda function triggered by an HTTP request made on the provisioned API Gateway REST API `/ping` or `/users` routes with `GET` method.
+
+### Locally
+
+In order to test the hello function locally, run the following command:
```

Comment:
Looks like a typo: should this be `test the ping function locally`?

Reply:
Yes good spot, it should be handler. In this demo there is only one function, called handler.